### DISCUSSION TITLE
robot force control from joint position trajectory.

### DISCUSSION THREAD

**Discussion by j-hae1** [NVlabs/curobo#184] [Software Q&A]
Thank you for good software curobo!

I am currently using curobo in isaacgym.
If it had been as it was, we would have used the robot joint position calculated from the curobo by setting the dof_target state, but unfortunately, we currently have to control the robot through force control.

How can I use robot joint trajectory for that force control? (There are too many actions that shake the robot while it is moving through the direct PID system, so it does not seem to work properly.)
In here meaning of force control is, i need the gain and damping too from the curobo

**Comment by balakumar-s** [comment]
cuRobo does not solve over dynamics. A standard PD controller should be able to track the trajectory from cuRobo. Do you have a base PD controller implemented in Isaac gym?
